CONTRIBUTING.md

```diff
@@ -60,18 +60,21 @@ This will automatically update go.mod and go.sum.
 Commit your changes, and the next time `./buildpack/conjur-env/build.sh` is run the
 `buildpack/vendor/conjur-env`directory will be created with updated dependencies.
 
-When upgrading the version of Go for `conjur-env`, both the pre-built 
-offline version and online version need to be
-updated:
-
-* **Offline build:** Update the base image version in `.buildpack/conjur-env/Dockerfile`
-
-* **Online build:** Update the version and file hashes in `manifest.yml`.
-  Available versions and hashes can be found [here][buildpacks],
-  or see the manifest for the [official Go Buildpack][go-buildpacks].
+When upgrading the version of Go for `conjur-env`, the value needs to be updated
+in a few places:
+
+* Update the base image in `.buildpack/conjur-env/Dockerfile
+* Update the Go version in `./buildpack/conjur-env/go.mod`
+* Update the version and file hashes in `manifest.yml` - available versions and
+  hashes can be found [here][buildpacks], or see the manifest for the
+  [official Go Buildpack][go-buildpacks]. (This is for the offline version of
```

Comment:
This was broken before this change, but `go-buildpacks` link should be `go-buildpack`.